XDR expired IOCs (#21437)
demisto/content: Packs/CortexXDR/ReleaseNotes/4_8_18.md

#### Integrations
##### Cortex XDR - IOC

> - Fixed an issue that when all indicators that follow the "Sync Query" are expired in XSOAR TIM, they will be removed from XDR IOCs.

Reviewer: ```suggestion
- Fixed an issue in the command **xdr-iocs-sync** where indicators were removed from XDR when all indicators that follow the "Sync Query" expired in XSOAR TIM.
```
